subprojects/docs/src/docs/userguide/javaPlugin.xml

```diff
@@ -375,10 +375,21 @@
                 </tr>
             </thead>
             <tr>
+                <td>implementation</td>
+                <td>api</td>
+                <td>-</td>
+                <td>
+                    Dependencies which are purely internal implementation details and not meant to be exposed to consumers.
+                </td>
+            </tr>
+            <tr>
                 <td>compile</td>
                 <td>-</td>
                 <td>-</td>
-                <td>Compile time dependencies</td>
+                <td>
+                    Compile time dependencies. This configuration is weakly deprecated in favor of the <code>implementation</code> configuration defined
+                    in <xref linkend="sec:java_library_configurations_graph"/>
```

Comment:
We should remove the reference to the java library plugin, `implementation` is described just a few lines above.